app/cdap/components/PipelineList/DraftPipelineView/DraftActions/index.tsx

```diff
@@ -157,6 +162,9 @@ class DraftActions extends React.PureComponent<IProps, IState> {
   public render() {
     return (
       <div className="action" onClick={(e) => e.preventDefault()}>
+        {this.state.errorMessage && (
+          <ErrorBanner error={this.state.errorMessage} canEditPageWhileOpen={true} />
```

Comment:
you probably wanna add an onClose method to reset the errorMessage upon closing the banner like:
```javascript
<ErrorBanner error={this.state.errorMessage} onClose={() => this.setState({ errorMessage: null })} canEditPageWhileOpen={true} />
```